### ISSUE TITLE
Low GPU utilization and general question

### ISSUE THREAD

**Issue by rvdsarr** [naiveHobo/InvoiceNet#63]
Hello guys!

I've been using the software to test it but I have some question. 
https://prnt.sc/zbqhz8

- I have a RTX2060 SUPER and I get results of 4.30 steps/sec onyl when using batch_size = 1. If I increase the batch size, the steps go down pretty fast. Is this normal behaviour? What does the steps even mean?

- My GPU utilization is only 1-3%. Is this normal? I get no errors when running the program and it says it using the GPU. I have installed Cudnn and all other depencies. 

- Can you clarify what train loss and val loss means? I could not find that anywhere

Thank you. I think its important to clarify this for other users aswell.

Kind regards, Roy
